config/locales/fr.yml

```diff
@@ -655,6 +655,7 @@ fr:
         labels:
           profile_picture: 'Photo de profile'
           remove_picture: 'Effacer la photo'
+          about: 'À notre propos'
```

Comment:
I think these locales should have the attention of a fluent speaker. For example _À notre propos_ translates as _About us_. Unless I'm mistaken this should be simply _À propos_ or possibly _À moi propos_. I'm not a native French speaker. (I believe this is also a problem with _Über uns_ as well).

Clearly this is an existing issue but excepting the rare cases where someone's preferred pronoun is plural I don't think that the label on the account edit page should be _About us_.

Reply:
It looks like we use this translation elsewhere: https://github.com/ifmeorg/ifme/blob/ff6c1bca9859ef575abe641cc34ea6c32bc4ba67/config/locales/fr.yml#L29, which was originally translated by French-speaking folks. I know French myself, and this translation is appropriate since it's an about page on our project.

Reply:
Clearly the translation is correct when appears in the navigation as it pertains to the project. I'm not suggesting the original usage there was incorrect. What I meant is that the translation which appears on the user's account edit page is still _À notre propos_. It seems strange to me to have the about section on the account edit page labeled _About Us_. Again I'm no native speaker but that strikes me as incorrect. Perhaps it is an idiomatic usage I'm not aware of.

For more context here is a screenshot of what I'm referring to.
![screenshot_2018-11-10 if me - compte](https://user-images.githubusercontent.com/1638226/48308893-890d9500-e53b-11e8-963e-9849e73cdde6.png)

Reply:
Aha, you're actually right about the above translation! Sorry for prematurely merging